package exception.test;

public class ClassA {

	public static void main(String[] args) throws Exception {
		// TODO 自動生成されたメソッド・スタブ
		ClassB cls = new ClassB();
		cls.funcB1();
	}

}
